operator/src/main/java/org/bf2/operator/KasFleetShardOperator.java

```diff
@@ -13,7 +17,23 @@
 import java.util.stream.Collectors;
 import java.util.stream.StreamSupport;
 
-public class KasFleetShardOperator implements QuarkusApplication {
+@CSVMetadata(
+        description = "Operator That Manages Kafka Instances",
+        displayName = "KaaS Fleetshard Operator",
+        provider = @Provider(
+                name = "bf2 community",
+                url = "https://github.com/bf2fc6cc711aee1a0c2a/kas-fleetshard"),
+        minKubeVersion = "1.21.0",
```

Comment:
I suspect it's higher than 1.21 by now. We could bump to 1.24 to align with OCP 4.11